web/src/app/escalation-policies/PolicyStepForm.js

```diff
@@ -103,6 +103,8 @@ export default class PolicyStepForm extends React.Component {
         classes={{
           badge: classes.badge,
         }}
+        tabIndex='0'
+        aria-label={`toggle ${txt}`}
```

Comment:
nit:
```suggestion
        aria-label={`Toggle ${txt}`}
```

since output would be `toggle Add Users`